projects/js-packages/components/components/record-meter-bar/index.tsx

```diff
@@ -32,7 +32,11 @@ export type RecordMeterBarProps = {
 	/**
 	 * The formatting style for legend item display. If not provided, it defaults to showing legend label after count
 	 */
-	showLegendLabelBeforeCount: boolean;
+	showLegendLabelBeforeCount?: boolean;
+	/**
+	 * The sort style for legend item. If not provided, it defaults to no sorting.
+	 */
+	sortByDescending?: boolean;
```

Comment:
What if someone wants to sort in ascending order?

Reply:
Such boolean props make it often confusing to decide what it actually does. So being explicit is better for readability. So, having a prop like `sortByCount?: 'ascending' | 'descending';` would be better in terms of readability, intent and control in the hands of consumer.

Reply:
That makes sense!